common-lib/src/main/scala/com/gu/mediaservice/lib/BaseStore.scala

```diff
@@ -43,8 +42,16 @@ abstract class BaseStore[TStoreKey, TStoreVal](bucket: String, config: CommonCon
 
   private var cancellable: Option[Cancellable] = None
 
-  def scheduleUpdates(scheduler: Scheduler) {
-    cancellable = Some(scheduler.scheduleAtFixedRate(0.seconds, 10.minutes)(() => update()))
+  def scheduleUpdates(scheduler: Scheduler): Unit = {
+    cancellable = Some(scheduler.scheduleAtFixedRate(0.seconds, 10.minutes)(() => {
+      try {
+        update()
+        lastUpdated.send(DateTime.now())
```

Comment:
previously 2 out of the 3 subclassing stores did this before trying the update (and the 3rd never sent an updated time). move it here from all of them and do at a sensible time (ie. _after_ the update has returned successfully) so that it makes sense